### ISSUE TITLE
Task list and discussion items for initial version release

### ISSUE THREAD

**Issue by lwjohnst86** [NetCoupler/NetCoupler#164]
I think we should submit a solid working version to CRAN soon. What I'm thinking is to focus entirely on the method and not any graphical functions. Those will come at a later point. For now, I'd like to agree that these functions are the initial "user-accessible" ones:

- `nc_standardize`
- `nc_estimate_network`
- `nc_estimate_exposure_links`
- `nc_estimate_outcome_links`

Nice to have, but not necessary right now are:

- `nc_plot_*` functions

If we agree on these, than the tasks to do before submitting to CRAN are:

- [x] Make diagram showing functions and their input and output
- [x] Make sure input/output of each main function is consistent and well documented.
- [x] Remove all other functions (not re-exports) to be unexported.
- [x] Update Getting Started documentation.
- [x] Make sure unit tests match I/O of diagram from above.
- [x] Prevent other function documentation from being generated with `@noRd`.
- [x] Add "examples using different methods" vignette with linear, logistic, and cox models (for now).

In addition to these tasks, [Milestone 1](https://github.com/NetCoupler/NetCoupler/milestone/1) tracks other issues that need to be completed before submitting the package.
